layouts/_default/baseof.html

```diff
@@ -28,16 +28,17 @@
     <title>
       {{ if .Title }}Galaxypedia - {{ .Title }}{{ else }}Galaxypedia{{ end }}
     </title>
-    <script
-      src="https://js.sentry-cdn.com/79a2541d247797be5c61ecfd7c307b0e.min.js"
-      crossorigin="anonymous"
-    ></script>
-    <script>
-      Sentry.onLoad(function() {
-        Sentry.init({
-          });
-      });
-    </script>
+    {{ if getenv "CI" }}
```

Comment:
tbh i'd rather use `hugo.IsDevelopment`

sure having it only run in CI is good but running in production *is* running in production. and the general assumption is that there are no issues in production, either locally or remotely

Reply:
I wasn't aware that was a thing, I'll change that now.